lib/state_machines/machine.rb

```diff
@@ -658,6 +658,12 @@ def dynamic_initial_state?
     #   directly to the object
     def initialize_state(object, options = {})
       state = initial_state(object)
+
+      unless object.respond_to? attribute
+        object.class.send(:alias_method, attribute, :state)
```

Comment:
Dropping it seem reasonable until someone find a way to fix it.  We will have to have a update the Changelog.